### PR TITLE
Feat/pt 9258

### PR DESCRIPTION
**Pull request by basilkot**
Description:

--

QA-test:

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.10.0-pr-327-ff57-ff5762f8.zip

### CODE REVIEW

File: client-app/shared/static-content/components/text-block.vue [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="pt-6 pb-16 text-block">
     <div class="w-full max-w-screen-2xl mx-auto px-5 md:px-12">
-      <h1 class="text-2xl mb-8">{{ model.header }}</h1>
+      <h2 v-if="model.heading == 'h2'" class="text-2xl">{{ model.title }}</h2>
+      <h3 v-if="model.heading == 'h3'" class="text-2xl">{{ model.title }}</h3>
+      <h4 v-if="model.heading == 'h4'" class="text-2xl">{{ model.title }}</h4>
+      <h5 v-if="model.heading == 'h5'" class="text-2xl">{{ model.title }}</h5>
+      <h6 v-if="model.heading == 'h6'" class="text-2xl">{{ model.title }}</h6>
```
Comment (artmakarov) [inline client-app/shared/static-content/components/text-block.vue:8]: can be on one line:
`<component :is="model.heading" class="text-2xl">{{ model.title }}</component>`

File: client-app/shared/static-content/components/index.ts [hunk 2]
```diff
@@ -2,23 +2,25 @@ import { Component } from "vue";
...
-  "side-image": SideImage,
+  "call-to-action-with-image": CallToActionWithImage,
   "promo-text": PromoText,
-  "column-images": ColumnImages,
-  "dark-promo-text": DarkPromoText,
+  features: Features,
+  "call-to-action": CallToAction,
   "subscribe-form": SubscribeForm,
   products: ProductsBlock,
+  title: TitleBlock,
```
Comment (artmakarov) [inline client-app/shared/static-content/components/index.ts:23]: maybe add postfix "-block" to all of them

File: client-app/shared/static-content/components/title-block.vue [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="pt-6 pb-16 lg:py-24">
+    <div class="w-full max-w-screen-2xl mx-auto px-5 md:px-12">
+      <h2 v-if="model.heading == 'h2'" class="text-2xl">{{ model.title }}</h2>
+      <h3 v-if="model.heading == 'h3'" class="text-2xl">{{ model.title }}</h3>
```
Comment (artmakarov) [inline client-app/shared/static-content/components/title-block.vue:5]: can be on one line:
`<component :is="model.heading" class="text-2xl">{{ model.title }}</component>`

File: client-app/shared/static-content/components/text-block.vue (hunk 1 above)
Comment (basilkot) [reply client-app/shared/static-content/components/text-block.vue]: fixed

File: client-app/shared/static-content/components/title-block.vue (hunk 3 above)
Comment (basilkot) [reply client-app/shared/static-content/components/title-block.vue]: fixed

File: client-app/shared/static-content/components/index.ts (hunk 2 above)
Comment (basilkot) [reply client-app/shared/static-content/components/index.ts]: added

**Review by artmakarov — APPROVED** [review]

**Pull request merged**